### ISSUE TITLE
Whether will ARCT affect other pre-trained model ?

### ISSUE THREAD

**Issue by LFhase** [IKMLab/arct2#3]
Hello, your work is really impressive. However, I am confused about belowing two points:
- whether this adversarial test influences other pre-trained models, like XLNet ? Did you do such tests before ?
- you also mention in your paper that BERT may diverge in small dataset. How can we know whether the model in experiments in ARCT converge or not? Maybe we should do some experiments in such small datasets and use your evluation methods to see the effect of dataset size. *Since I am a newbie of NLP, maybe it's a kind of trivial question.*

Many Thanks and Best : )
LFhase

**Comment by timniven** [comment]
Hi LFhase,

We haven't tested XLNet. A broader question though is why BERT can't solve this task, and whether XLNet is likely to have whatever BERT lacks? I think it is important to develop an intuition about this. Of course, you are welcome to conduct this experiment (and let me know the results!) since it actually doesn't cost very much to just try. But since my intuition is that XLNet is very unlikely to have the world knowledge needed for the task, I do not expect it to work, and therefore don't plan to conduct the experiment myself. However, I welcome you to prove me wrong.

The degenerate runs on small training sets are discussed in the original paper (I don't think you would call it "divergence," but rather a lack of good convergence - a "degenerate run" is what the original authors call it). In our case it is actually a rather subjective judgment. Looking at the training accuracies of BERT's runs, you can generally see that when BERT doesn't get over 80% on the training set it performs poorly on the validation and or test sets. I'm not 100% sure why this happens, it could be that with such a small dataset there are more local minima to get stuck in during optimization. Again, if you can develop your own intuition about this kind of question, then hopefully you can design an experiment to test your hypothesis. But we do not suggest using the original dataset anymore because of the bias coming from uneven distributions of linguistic artifacts over the labels. Since all models love to exploit these statistics, this is a meaningless exercise. What we have called the "adversarial" dataset (which may be not have been the best choice of words) is what you should use for any future work on ARCT.

Good luck with your studies and best wishes to you :)

Tim.
